### ISSUE TITLE
[Bug Report][3.3.12] Impossible to use `fit-content` sizing for `VImg`

### ISSUE THREAD

**Issue by kappaflow** [vuetifyjs/vuetify#18019]
### Environment
**Vuetify Version:** 3.3.12
**Last working version:** 3.3.6
**Vue Version:** 3.3.4
**Browsers:** Chrome 115.0.0.0
**OS:** Windows 10

### Steps to reproduce
Add  `max-height="fit-content"`or/and `max-width="fit-content"` to `VImg`. It won't affect the sizing (the image will have zero size).

### Expected Behavior
The image suppose to have the maximum render size of the image (intrinsic size).

### Actual Behavior
It has a zero size (invisible)

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqNUctOwzAQ/JXI5zoWj1MVEEj8BeHgJtvEwXYs7yYtQv131imlIQi1t2RndmZ2/PopnkPIxwHEWhQELlhN8Fj6LCtqM2aV1YgPpajl1sI+6wYks/2QGHQFUsd+8HWmrWm8rMATxGzTyCbCh+RZS+DlfSkmtYWe0zNkgRHsSeKwIUMW5C3TXmCrB0uFYtp5Z5TGNae/7GedE+xaQ1CKM+b0XraQIjFha0hWPaf1tOTsTE3t/xSMFYMtUcC1UlXtU3GpkA7zqnfKON0AqqCjtlbvleMuo9E270Jz1nks1BT91MvPVaevQs0eQqyENf4d8w57z4/0mZilqBBLsc5ejxq/Q3VYgzVjzD2Q8sGp75RPd/ldfnPLLkinmbR6w+GTWpJ6K/2BHY0LfSTpdFjYHoHJepqkGSul/wsZnpim4uDJOJB176Yw98css3EO6OQm9juEyCKlWM1sUuArrC6eyx5/tBUbjhBlBF9DhHjtSYu1+VkL6M9pyf2QCj+sxLTGzR8zi7cvJ9gxJA==)

### Other comments
After #17090 commit `VImg` became absolute positioned, so its impossible to use the image intrinsic size. Also, there is no way to use API to make it work as before with relative positioning. There is no quick fix to make images to use their original (intrinsic) size as well... a lot of css changes must be hardcoded to overwrite the current `VImg` size implementation correctly. Please make it work as it used to...

**Comment by Lozoute** [comment]
Have the same problem here.
It seems that by default every VImg will be invisible (zero size) although we would expect VImg to have image natural (intrinsic) size by default instead.
Currently I have to hard code width and height to see my images appear but it is a problem since I don't always know the size of my image. I want to apply max-height and max-width to be sure that my display don't turn to sh*t but having to put width/height or min-width/height is a big problem.

**Comment by kappaflow** [comment]
Yes, I think its a very serious issue. Because of that, now I am stuck on v. 3.3.6, until it will be fixed somehow...

**Comment by kappaflow** [comment]
@KaelWD do you have any plans on fixing this issue in near future? Based on that, I will be able to decide if I can postpone Vuetify update for a few months or I might need to specify image sizes manually for almost every image on the website to restore the previous functionality... I really hope you revert it back again, or provide an option to decide what sizing for VImg to use...

**Comment by NitroG42** [comment]
A coworker found a global workaround for now, not sure if it has side effect but you can put 
```
.v-img__img {
  position: unset;
}
```
as a global style to have the previous behavior again. If you have any issue with this don't hesitate to inform us here.

**Comment by Estheisia** [comment]
Hi, I replaced to `<img>` instead of using the VImg, in my layout it worked with no other changes :

```
<v-app-bar>
  <img
    style="max-width:140px; max-height: 32px"
    :src="appBarImage"
  />
  <!-- ...restOfTheAppbar -->
</v-app-bar>
```

----- EDIT
Latest 3.12.0 changed VImg a bit :
VImg: add image-class prop (https://github.com/vuetifyjs/vuetify/issues/22622) ([7a34cfa](https://github.com/vuetifyjs/vuetify/commit/7a34cfa6fa79692d46e1056c5f2b1ae1603deca0)), closes https://github.com/vuetifyjs/vuetify/issues/17229
VImg: support width="fit-content" to adjust container width to image ratio (https://github.com/vuetifyjs/vuetify/issues/21414) ([1be6782](https://github.com/vuetifyjs/vuetify/commit/1be6782c5d8df5a4439c04714382777318070ea7))
